README.md

```diff
@@ -3,101 +3,105 @@
-  - Workaround: define types as globals in `.eslintrc` or define types and import them `import type ReactElement from './types'`
-- `no-unused-vars/no-undef` with Flow declarations (`declare module A {}`) [#132](https://github.com/babel/babel-eslint/issues/132#issuecomment-112815926)
-
-Modules/strict mode
-- `no-unused-vars: [2, {vars: local}]` [#136](https://github.com/babel/babel-eslint/issues/136)
-
-Please check out [eslint-plugin-react](https://github.com/yannickcr/eslint-plugin-react) for React/JSX issues
-- `no-unused-vars` with jsx
-
-Please check out [eslint-plugin-babel](https://github.com/babel/eslint-plugin-babel) for other issues
+Note that you only need to use babel-eslint if you are using Babel to transform your code. If this is not the case, please use the relevant parser for your chosen flavor of ECMAScript (note that the default parser supports all non-experimental syntax as well as JSX).
 
 ## How does it work?
 
-ESLint allows custom parsers. This is great but some of the syntax nodes that Babel supports
-aren't supported by ESLint. When using this plugin, ESLint is monkeypatched and your code is
-transformed into code that ESLint can understand. All location info such as line numbers,
+ESLint allows for the use of [custom parsers](https://eslint.org/docs/developer-guide/working-with-custom-parsers). When using this plugin, your code is parsed by Babel's parser (using the configuration specified in your [Babel configuration file](https://babeljs.io/docs/en/configuration)) and the resulting AST is
+transformed into an [ESTree](https://github.com/estree/estree)-compliant structure that ESLint can understand. All location info such as line numbers,
 columns is also retained so you can track down errors with ease.
 
-Basically `babel-eslint` exports an [`index.js`](/lib/index.js) that a linter can use.
-It just needs to export a `parse` method that takes in a string of code and outputs an AST.
+Note that ESLint's core rules do not support experimental syntax and may therefore not work as expected when using babel-eslint. Please use the companion [`eslint-plugin-babel`](https://github.com/babel/eslint-plugin-babel) plugin for core rules that you have issues with.
 
 ## Usage
 
-### Supported ESLint versions
-
-ESLint | babel-eslint
------------- | -------------
-4.x | >= 6.x
-3.x | >= 6.x
-2.x | >= 6.x
-1.x | >= 5.x
-
-### Install
-
-Ensure that you have substituted the correct version lock for `eslint` and `babel-eslint` into this command:
+### Installation
 
 ```sh
-$ npm install eslint@4.x babel-eslint@8 --save-dev
+$ npm install eslint babel-eslint --save-dev
 # or
-$ yarn add eslint@4.x babel-eslint@8 -D
+$ yarn add eslint babel-eslint -D
 ```
 
+**Note:** babel-eslint requires `babel/core@>=7.2.0` and a valid Babel configuration file to run. If you do have this already set up, please see the [Babel Usage Guide](https://babeljs.io/docs/en/usage).
```

Comment:
Should this say `don't have this already setup` rather than `do have this already setup`?

Reply:
:+1:

Reply:
Nice catch - thanks!